<?xml version="1.0"?>

<launch>
  <param name="/use_sim_time" value="true" />
  
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" 
    args="0.15 0.0 0.0 0.0 0.0 0.0 /base_link /laser 40" />

  <node pkg="karto_ros" type="karto_ros" name="karto_ros" output="screen">
    <remap from="scan" to="scan" />
    <param name="odom_frame" value="odom"/>
    <param name="map_update_interval" value="25"/>
    <param name="resolution" value="0.025"/>
    <rosparam command="load" file="$(find karto_ros)/config/mapper_params.yaml" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" />

</launch>
